tests/cli/workflow/nf_analysis/test_cli_workflow_base.py

```diff
@@ -5,6 +5,7 @@
 
 from cg.cli.workflow.base import workflow as workflow_cli
 from cg.constants import EXIT_SUCCESS, Workflow
+from cg.constants.nextflow import NFX_WORKFLOWS
```

Comment:
You are right, it is not. I leave the conversation open because it will depend on what we do with https://github.com/Clinical-Genomics/cg/pull/3128/#discussion_r1575138936

Reply:
Used NEXTFLOW_WORKFLOWS_DEV